stl/src/vector_algorithms.cpp

```diff
@@ -1396,6 +1420,58 @@ namespace {
         return _Find_trivial_tail(_First, _Last, _Val);
     }
 
+    template <class _Traits, class _Ty>
+    const void* __stdcall __std_find_last_trivial(const void* _First, const void* _Last, _Ty _Val) noexcept {
+        const void* const _Real_last = _Last;
+#ifndef _M_ARM64EC
+        size_t _Size_bytes = _Byte_length(_First, _Last);
+
+        const size_t _Avx_size = _Size_bytes & ~size_t{0x1F};
+        if (_Avx_size != 0 && _Use_avx2()) {
+            _Zeroupper_on_exit _Guard; // TRANSITION, DevCom-10331414
+
+            const __m256i _Comparand = _Traits::_Set_avx(_Val);
+            const void* _Stop_at     = _Last;
+            _Rewind_bytes(_Stop_at, _Avx_size);
+            do {
+                _Rewind_bytes(_Last, 32);
+                const __m256i _Data = _mm256_loadu_si256(static_cast<const __m256i*>(_Last));
+                const int _Bingo    = _mm256_movemask_epi8(_Traits::_Cmp_avx(_Data, _Comparand));
+
+                if (_Bingo != 0) {
+                    const unsigned long _Offset = _lzcnt_u32(_Bingo);
+                    _Advance_bytes(_Last, _Offset ^ 0x1F - (sizeof(_Ty) - 1));

```

Comment:
This is mixing bitwise `^` with arithmetic `-`, so our stylistic conventions demand parentheses here. In fact, because arithmetic `-` has much higher precedence than bitwise `^`, this looks like a bug.

Reply:
Even if it's not a bug, we need parentheses to make it clear that this is intentional.

Reply:
`^` reverted, DevCom-10487304 reported instead.